wd_helper.au3

```diff
@@ -843,6 +843,8 @@ Func _WD_FrameList($sSession, $bReturnAsArray = True, $iDelay = 1000, $iTimeout
 	Local $a_Result[0][$_WD_FRAMELIST__COUNTER], $sStartLocation = '', $sMessage = ''
 	Local $vResult = '', $iErr = $_WD_ERROR_Success, $iFrameCount = 0
 
+	_WD_LoadWait($sSession, $iDelay, $iTimeout, Default, $_WD_READYSTATE_Complete) ; first _WD_LoadWait with declared $iDelay
```

Comment:
`_WD_LoadWait` will be executed by "internal" routine, so why not use `__WD_Sleep($iDelay)` instead?

Reply:
> internally is used after _WD_FrameEnter was used so where is proper waiting for main document ?

Please confirm, but I believe that the very first execution of `_WD_FrameEnter` switches to the main document because `$sLevel` is "null". So there isn't a need to separately do this for the main document.

Also, you have no way of knowing the starting location (main document or frame) so this `_WD_LoadWait` isn't doing what you intended IMO.

> Sleep is not enough

I don't see why it doesn't work here given my explanation above.

Reply:
oh ... 
Now I see you have right... new  commit will be commited soon

Reply:
Please take a look: [__WD_Sleep($iDelay)](https://github.com/Danp2/au3WebDriver/pull/456/commits/37341d0d3b0e69ceb3695b7a914cce0225f5c123)